spec/integrations/mongo_spec.rb

```diff
@@ -0,0 +1,292 @@
+# encoding: utf-8
+require 'spec_helper'
+
+describe 'Bugsnag::MongoBreadcrumbSubscriber', :order => :defined do
+  before do
+    unless defined?(::Mongo)
+      @mocked_mongo = true
+      module ::Mongo
+        module Monitoring
+          COMMAND = 'command'
+          class Global
+          end
+        end
+      end
+      module Kernel
+        alias_method :old_require, :require
+        def require(path)
+          old_require(path) unless path == 'mongo'
+        end
+      end
+    end
+  end
+
+  it "should subscribe to the mongo monitoring service" do
+    expect(::Mongo::Monitoring::Global).to receive(:subscribe) do |*args|
```

Comment:
At the point this is run, the `::Bugsnag::MongoBreadcrumbSubscriber` doesn't exist, so I've split the difference by leaving it as a block and checking `command` and `subscriber` separately, but making an `instance_of` call rather than checking `respond_to?`s.